packages/react/src/components/Header/HeaderAction/HeaderActionPanel.jsx

```diff
@@ -96,12 +97,31 @@ const HeaderActionPanel = ({
       >
         <ul aria-label={item.label}>
           {item.childContent.map((childItem, k) => {
-            const ChildElement = childItem?.metaData?.element || 'a';
+            const { element, ...metaData } = childItem?.metaData ?? {};
+            const ChildElement = element || 'a';
+            const onKeyDownClick = (e) => e.target.click();
+
+            // otherwise if the item is an A and doesn't have an onClick event
```

Comment:
nit: comment starting with otherwise, was the code refactored and you forgot the comment?

Reply:
yeah, refactored and missed.